net.granjow.slowmovideo.yaml

```
@@ -0,0 +1,84 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+x-add-extensions:
+    org.freedesktop.Platform.ffmpeg-full:
+        directory: lib/ffmpeg
+        version: "21.08"
+        add-ld-path: "."
+
+cleanup-commands:
+    - mkdir -p /app/lib/ffmpeg
+
+command: slowmoUI
+rename-icon: AppIcon
+rename-desktop-file: slowmoUI.desktop
+#rename-appdata-file: hugin.appdata.xml
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  # Because the file chooser somehow doesn't show the files on the host, we export some directory for reading pcap files
+  - --filesystem=xdg-public-share
+
+
+modules:
+
+  - name: opencv
+    # Taken from https://github.com/flathub/org.d3cod3.Mosaic/blob/be9aea900361940120341b2ad6fe7d97bec39c66/org.d3cod3.Mosaic.yml#L152
+    buildsystem: cmake-ninja
+    builddir: true
+    cleanup:
+        - '/include/'
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Debug
```

```suggestion
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

A debug build is unlikely what you want.

---

ah, that applies to Wireshark as well, I filed a bug to not forget: https://github.com/flathub/org.wireshark.Wireshark/issues/93